python/pyspark/sql/functions/builtin.py

```diff
@@ -15427,15 +15443,20 @@ def map_contains_key(col: "ColumnOrName", value: Any) -> Column:
 
     Examples
     --------
-    >>> from pyspark.sql.functions import map_contains_key
+    Example 1: The key is in the map
+
+    >>> from pyspark.sql import functions as sf
     >>> df = spark.sql("SELECT map(1, 'a', 2, 'b') as data")
-    >>> df.select(map_contains_key("data", 1)).show()
+    >>> df.select(sf.map_contains_key("data", 1)).show()
     +-------------------------+
     |map_contains_key(data, 1)|
     +-------------------------+
     |                     true|
     +-------------------------+
-    >>> df.select(map_contains_key("data", -1)).show()
+
+    Example 2: The key is not in the map
+
+    >>> df.select(sf.map_contains_key("data", -1)).show()
```

Comment:
```suggestion
    >>> from pyspark.sql import functions as sf
    >>> df = spark.sql("SELECT map(1, 'a', 2, 'b') as data")
    >>> df.select(sf.map_contains_key("data", -1)).show()
```

Reply:
Do we need to make each example self-contained now?
I remember writing this way before, it was suggested not to write duplicate but reusable code in every example.

Reply:
I prefer to `self-contained`, cc @zhengruifeng

Reply:
Actually, I can do it all, but I think it's best for us to have `a stable rule`.

Reply:
cc @HyukjinKwon for the above question about `from pyspark.sql import functions as sf`
> Do we need to make each example self-contained now?

Reply:
yeah. we have discussed in some other PRs. Should make it self-contained with `from pyspark.sql import functions as sf`.

Reply:
Thank you for informing us the guidance, @HyukjinKwon . cc @panbingkun